AEPMessaging/Sources/Feed.swift

```diff
@@ -0,0 +1,27 @@
+/*
+ Copyright 2023 Adobe. All rights reserved.
+ This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License. You may obtain a copy
+ of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software distributed under
+ the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ OF ANY KIND, either express or implied. See the License for the specific language
+ governing permissions and limitations under the License.
+ */
+
+import AEPServices
+import Foundation
+
+@objc(AEPFeed)
+@objcMembers
+public class Feed: NSObject, Codable {
+    /// Identification for this feed, represented by the AJO Surface URI used to retrieve it
+    public var id: String
```

Comment:
Should we rename this to `surface` instead? Also should this contain the full surface URI or only the customer provided surface path? I'm wondering if the base should be opaque to the customer because 1) They do not provide it 2) When we return [String: Feed] to them, the surface value should be same (consistent) path value they provide in the dictionary key as well as in the Feed->surface (or id).

Reply:
I had `surfaceUri` on my mind as well, however if we do not include the base (mobileapp://<app bundle identifier>) then it's more of a path (promos/feed1) than URI. 
Also, regarding `i'd also expect the key value to be a full surface uri as returned by konductor.` it would imply - the customer provides us surface paths ["promos/feed1", "promos/feed2"] and when they try to retrieve the content ([String: Feed]) for the same paths, they now need to know and use the Uri instead which is non-intuitive.